sonar-auth-saml-plugin/src/main/java/org/sonarsource/auth/saml/SamlSettings.java

```diff
@@ -102,14 +102,6 @@ boolean isEnabled() {
       configuration.get(USER_NAME_ATTRIBUTE).isPresent();
   }
 
-  @CheckForNull
-  private static String urlWithEndingSlash(@Nullable String url) {
-    if (url != null && !url.endsWith("/")) {
-      return url + "/";
```

Comment:
In fact if we don't want to have a last "/" in this URL, we should prevent it and instead of always adding it if missing, we should always remove it if it exist

Reply:
No, because some IDP *might* require the HTTP-Redirect URL to have a slash last. We don't know that. We should trust that the user inputs the correct URL. If the user doesn't we shouldn't second guess them.